### PR TITLE
artifacts: Add docs for file/artifact promotion

### PR DESCRIPTION
**Pull request by justaugustus**
(Part of https://github.com/kubernetes/k8s.io/issues/2624, https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/413.)
Supports https://github.com/kubernetes/k8s.io/pull/2272.

Signed-off-by: Stephen Augustus <foo@auggie.dev>

### CODE REVIEW

**Comment by justaugustus** [comment]
/assign @ameukam @spiffxp @dims @justinsb 
/cc @puerco @cpanato @Verolop @saschagrunert @jingxu97 
cc: @kubernetes-sigs/release-engineering @kubernetes/k8s-infra-group-admins @kubernetes/wg-k8s-infra-leads

File: artifacts/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Kubernetes Artifacts
+
+This directory the manifest that are used to perform artifact promotion for
+the Kubernetes project.
```
Comment (ameukam) [inline artifacts/README.md:4]: 🤔  I'm not sure about the meaning of this (forgive my poor English). Maybe something like : 
```suggestion
This directory contains the manifests that are used to perform artifact promotion for
the Kubernetes project.
```

Comment (justaugustus) [reply artifacts/README.md]: @ameukam -- Nice catch! The wording was off and I've pulled in your suggestion. :)

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by dims** [comment]
/approve

leaving `/lgtm` for a sig-release reviewer.

**Pull request merged**